Documentation/reports.md

```diff
@@ -241,7 +241,33 @@ spec:
   reportingEnd: "2019-07-31T00:00:00Z"
 ```
 
+### expiration 
+
+Add the expiration field to set a retention period on a scheduled metering Report. You can avoid manually removing the Report by setting the expiration duration value. The retention period is equal to the Report creationDate plus the expiration duration. The Report is removed from the cluster at the end of the retention period if no other Reports or ReportQueries depend on the expiring Report. Deleting the Report from the cluster can take several minutes.
```

Comment:
It seems like we use backticks to highlight parameter names, so:
```suggestion
Add the expiration field to set a retention period on a scheduled metering Report. You can avoid manually removing the Report by setting the expiration duration value. The retention period is equal to the Report creationDate plus the `expiration` duration. The Report is removed from the cluster at the end of the retention period if no other Reports or ReportQueries depend on the expiring Report. Deleting the Report from the cluster can take several minutes.
```

Reply:
Yeah, I would agree with this as well. The OCP documentation set uses the IBM styling guidelines, but we're rather consistent on backticking things like the metering-related custom resources (e.g. `Report`, `ReportDataSource`, etc.) and fields that belong to those objects (e.g. `metadata.name`, `spec.expiration`, etc.).